config/routes.rb

```diff
@@ -1,6 +1,12 @@
 Rails.application.routes.draw do
+
   root to: redirect("/#{I18n.default_locale}"), as: :redirected_root
   scope "(:locale)", locale: /#{I18n.available_locales.join("|")}/ do
+
+    devise_for :users do
+      get '/users/sign_out', to: 'devise/sessions#destroy'
```

Comment:
Why we need custom rout here? And why we are using get to destroy users session. FOr me it should be delete method instead of get.